src/client/grid/game_map.tsx

```diff
@@ -116,6 +113,24 @@ function confirmDeliveryCb(
   };
 }
 
+export function getConfirmDeliveryMessage(
+  player: PlayerColor | undefined,
+  moveAction: MoveData,
+  grid: Grid,
+  moveInstance: MoveAction,
+) {
+  const endingStop = grid.get(peek(moveAction.path).endingStop) as City;
+  const income = moveInstance.calculateIncome(moveAction);
+  const counts = [...income]
+    .filter(([a]) => a != null)
```

Comment:
I've been meaning to fix this, can you double check that income is != 0 too? (Don't check > 0 on the off chance that negative income may be earned lol).